testlib/doc.go

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-// testlib exposes helper methods for running unit tests against a containerized test store.
+// Package testlib exposes helper methods for running unit tests against a containerized test store.
 package testlib
```

Comment:
Is it possible to fix the remaining issues in `testlib` without breaking another package?
```
$ golangci-lint run ./testlib/...
testlib/cluster.go:24:32: method GetClusterId should be GetClusterID (golint)
func (c *FakeClusterInterface) GetClusterId() string { return "" }
                               ^
testlib/helper.go:24:2: struct field `SqlSupplier` should be `SQLSupplier` (golint)
	SqlSupplier      *sqlstore.SqlSupplier
	^
testlib/helper.go:135:22: method GetSqlSettings should be GetSQLSettings (golint)
func (h *MainHelper) GetSqlSettings() *model.SqlSettings {
                     ^
testlib/helper.go:151:22: method GetSqlSupplier should be GetSQLSupplier (golint)
func (h *MainHelper) GetSqlSupplier() *sqlstore.SqlSupplier {
                     ^
```

Reply:
Unfortunately it breaks other package because it's intended to mock `einterfaces.ClusterInterface` which also has `GetClusterId()`.

```
# github.com/mattermost/mattermost-server/v5/api4 [github.com/mattermost/mattermost-server/v5/api4.test]
api4/plugin_test.go:265:20: cannot use testCluster (type *testlib.FakeClusterInterface) as type einterfaces.ClusterInterface in assignment:
	*testlib.FakeClusterInterface does not implement einterfaces.ClusterInterface (missing GetClusterId method)
		have GetClusterID() string
		want GetClusterId() string
FAIL	github.com/mattermost/mattermost-server/v5/api4 [build failed]
```

I'll fix the remaining issues which don't break.

Reply:
Now the remaining issue is 

```
testlib/cluster.go:24:32: method GetClusterId should be GetClusterID
```

The remaining warnings are just `Exported method / type should have comment`.

Reply:
Got it. Then lets leave `testlib` as it is for now.